docs/interactions/Slash_Commands.md

```diff
@@ -795,6 +861,23 @@ If you specify `choices` for an option, they are the **only** valid values for a
 | name  | string        | 1-100 character choice name                     |
 | value | string or int | value of the choice, up to 100 characters if string |
 
+## ApplicationCommandPermissions
+
+Application command permissions allow you to enable or disable commands for specific users or roles within a guild.
+
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| id    | snowflake | the id of the role or user |
+| type | [ApplicationCommandPermissionType](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommandpermissiontype) | role or user |
+| permission | boolean | `true` to allow, `false`, to disallow |
+
+## ApplicationCommandPermissionType
+
+| Name | Value |
+| -----| ----- |
+| ROLE | 1     |
+| USER | 2     |
```

Comment:
Should this be a `0` and `1` for just general consistency (see: channel overrides/overwrites/whatever they are called.)